### PR TITLE
fix: adapt to ani-skip update

### PR DESCRIPTION
**Pull request by Woreec**
Ani-Skip has been updated, with changed start arg

# Pull Request Template

## Type of change
Ani-Skip has been updated, with changed start arg, so tiny fix for that.

- [x] Bug fix
- [ ] Feature
- [ ] Documentation update

## Description

I hope this is okay this way.
start arg now is -q [title] -e [episode]

## Checklist

- [x] any anime playing
- [x] bumped version
---
- [x] next, prev and replay work
- [x] `-c` history and continue work
- [x] `-d` downloads work
- [ ] `-s` syncplay works
- [x] `-q` quality works
- [x] `-v` vlc works
- [x] `-e` select episode works
- [ ] `-S` select index works
- [x] `-r` range selection works
- [x] `--skip` ani-skip works
- [x] `--skip-title` ani-skip title argument works
- [ ] `--no-detach` no detach works
- [x] `--dub` and regular (sub) mode both work
- [ ] all providers return links (not necessarily on a single anime, use debug mode to confirm)
---
- [ ] `-h` help info is up to date
- [ ] Readme is up to date
- [ ] Man page is up to date

## Additional Testcases

- The safe bet: One Piece
- Episode 0: Saenai Heroine no Sodatekata ♭
- Unicode: Saenai Heroine no Sodatekata ♭
- Non-whole episodes: Tensei shitara slime datta ken (ep. 24.5, ep. 24.9)

### CODE REVIEW

**Comment by synacktraa** [comment]
There's a reason why I introduced getting MyAnimeList Id directly from ani-skip, cache it in `mal_id` variable when parsing cli arguments.

Line no. 424 should be:

```python

[ "$skip_intro" = 1 ] && (dep_ch "ani-skip" || true) && mal_id=$(ani-skip -q "${skip_title:-${title}}")
```

In `play_episode` function, it should be:

```python
[ "$skip_intro" = 1 ] && skip_flag="$(ani-skip -q "$mal_id" -e "$ep_no")"
```

> One can pass MyAnimeList ID directly to `--skip-title` and it would still work!

**Comment by synacktraa** [comment]
> Ahhh, i see. I really should have just waited for you to do that, sorry about that i got a bit overzealous... I changed it to that now.
> 
> 

No problem. I hope you tested it once after making changes.

**Comment by Woreec** [comment]
I did... but something is still odd actually it doesnt work with every anime. let me make it a draft again.. i feel so embarassed right now whew, sorry if im causing issues :^)

**Comment by Woreec** [comment]
Yea, i figured out what the issue was i had, totally unrelated to this, had to do with my internet.
I tested it now with One piece, frieren and some more and they all work perfecly fine, including using --skip_title
So it really should work perfecly now.

**Comment by port19x** [comment]
I took the liberty to reword your PR title. Thanks for submitting

**Comment by port19x** [comment]
Also bump the patch version please

File: ani-cli [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-version_number="4.8.3"
+version_number="4.8.4"
```
Comment (port19x) [inline ani-cli:3]: Merged another PR before yours, so we'll have to re-bump
```suggestion
version_number="4.8.5"
```

File: ani-cli [hunk 2]
```diff
@@ -265,9 +265,8 @@ download() {
     esac
 }
 
-play_episode() {
-    aniskip_title="${skip_title:-${title}}"
-    [ "$skip_intro" = 1 ] && skip_flag="$(ani-skip "$aniskip_title" "$ep_no")"
+play_episode() {
```
Comment (port19x) [inline ani-cli:268]: Didn't see the other whitespace
```suggestion
play_episode() {
```

**Review by port19x — APPROVED** [review]

**Pull request merged**